app/controllers/application.js

```diff
@@ -11,6 +12,24 @@ import OSFAgnosticAuthControllerMixin from 'ember-osf/mixins/osf-agnostic-auth-c
  * @extends Ember-OSF.OSFAgnosticAuthControllerMixin
  */
 export default Ember.Controller.extend(OSFAgnosticAuthControllerMixin, {
+    i18n: Ember.inject.service(),
     toast: Ember.inject.service(),
     theme: Ember.inject.service(),
+    init() {
+        const documentType = preprintWord();
+        const i18n = this.get('i18n');
+
+        let preprintWords = {};
+        Object.keys(documentType).forEach(function(key1) {
+            const entry = documentType[key1];
+            Object.keys(entry).forEach(function(key2) {
+                const word = entry[key2];
+                preprintWords[`${word}`] = i18n.t(`documentType.${key1}.${key2}`);
+            });
+        });
+
+        this.get('i18n').addGlobals({preprintWords: preprintWords});
```

Comment:
This seems like it belongs in `theme`, so whenever the provider changes, the preprint translations change as well.